release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -225,6 +256,10 @@ link:https://github.com/openshift/cluster-monitoring-operator[*cluster-monitorin
 As an {product-title} administrator, you can view system resources, containers,
 and components metrics in one dashboard interface, Grafana.
 
+In {product-title} 3.11, the Cluster Monitoring Operator installation is enabled
+by default. Ensure there is an available node in your cluster to avoid
+unexpected failures.
+
```

Comment:
Adding more clarification: By default, "Cluster Monitoring Operator" install is expecting 'node-role.kubernetes.io/infra=true' node in your cluster, you also could change it by setting openshift_cluster_monitoring_operator_node_selector in inventory file to your customized node selector.

Reply:
Updated. Thanks!